.github/workflows/release-tagger.yml

```
@@ -0,0 +1,37 @@
+    branches:
+      - 'dev'
+
+jobs:
+  create-tag:
+    runs-on: ubuntu-latest
+    env:
+      REGEX_PATTERN: '([0-9]+\.[0-9]+(\.[0-9]+)?)$'
+    steps:
+      - name: Check PR title with regex
+        id: check_pr_title
+        run: |
```

actionlint (1.7.4)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)